docs/howtos/consuming-rust-components-on-ios.md

```diff
@@ -1,10 +1,37 @@
-## Guide to Consuming Rust Components on iOS
+# Guide to Consuming Rust Components on iOS
+
+The application services libraries are published as a single zip file containing all the individual component frameworks (such as *Logins.framework*, *FxAClient.framework*) and also a single composite (megazord) framework called *MozillaAppServices.framework* containing all the components.
+
+The client-side can choose to use a single component framework, or the composite.
+
+The package is published as a release on github: https://github.com/mozilla/application-services/releases
+
+## Carthage
+
+- Add the dependency line to the Cartfile, for instance: `github "mozilla/application-services" ~> "v0.16.1"` 
+- `carthage` will download MozillaAppServices.frameworks.zip, and add all the available frameworks to the 'Carthage/' dir.
+- Choose which framework to link against for your project (in the *Link Binary with Libraries* step in your Xcode target).
+- Add additional dependencies, see [below](#additional-dependencies).
+
+### Adding a carthage provided framework to Xcode
+- In general, to do this, add *XXX.framework* from *Carthage/Build/iOS* to *Link binary with Libraries* for the Xcode target, and also to the *Copy Carthage Dependencies* step
+
+### Using a Circle-CI built framework
+
+Rather than using a tagged release version, one can grab the build from Circle-CI, like so:
```

Comment:
It would have been nice to add more details like where to find this copy-pasta line. It's fine let's just do it in a future PR.

Reply:
Yes lets update these docs as more info becomes available (I recommend the handy edit-in-place button in Github which will create a PR, and ping me for review), I would like to know how to generate that line.